src/features/groups/components/ContainerCardList/ContainerCard/ContainerCardDropdownMenuTriggerButton/ContainerCardDropdownMenuContent/DeleteContainerDialogTriggerButton/DeleteContainerDialogContent.tsx

```diff
@@ -0,0 +1,54 @@
+import {
+  Button,
+  DialogBody,
+  DialogClose,
+  DialogContent,
+  DialogFooter,
+  DialogHeader,
+  DialogTitle,
+} from '@/components/ui';
+
+interface IDeleteContainerDialogContentProps {
+  /**
+   * The function to close parent UI which is dropdownMenu
+   */
+  onParentClose: () => void;
+  /**
+   * The function to close this dialog
+   */
+  onDialogClose: () => void;
+}
+
+export const DeleteContainerDialogContent = ({
+  onParentClose,
+  onDialogClose,
+}: IDeleteContainerDialogContentProps) => {
+  /**
+   * The function to handle when delete button clicked
```

Comment:
Please describe in the TSDoc that this function closes both the parent and this dialog.
To ensure we remember it when you add functionality in the next PR.